rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNames/rename_class_with_same_name_but_different_namespace_in_class_property.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureAutoImportNames;
+
+use Rector\Tests\Renaming\Rector\Name\RenameClassRector\Source\OldClass;
+
+class SomeClass
+{
+    /**
+     * @var OldClass
+     */
+    private $class;
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureAutoImportNames;
+
+use Rector\Tests\Renaming\Rector\Name\RenameClassRector\Source\NewClass;
+
+class SomeClass
+{
+    /**
+     * @var NewClass
+     */
+    private $class;
+}
+
+?>
```

Comment:
`OldClass` and `NewClass` is different name, please change to:

```suggestion
<?php

namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureAutoImportNames;

use Rector\Tests\Renaming\Rector\Name\RenameClassRector\Source\FirstNamespace\SomeServiceClass;

class SomeClass2
{
    /**
     * @var SomeServiceClass
     */
    private $someService;
}

?>
-----
<?php

namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureAutoImportNames;

use Rector\Tests\Renaming\Rector\Name\RenameClassRector\Source\FirstNamespace\SomeServiceClass;

class SomeClass2
{
    /**
     * @var SomeServiceClass
     */
    private $someService;
}

?>

```

Reply:
Done